tools/packages.config

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+<? xml version="1.0" encoding="utf-8"?>
```

Comment:
As far as I know this is no longer valid XML

Reply:
@Jericho Would it be possible to fix this in the tool?

Reply:
For sure, I’ll take care of it today.

Reply:
Done.